Source/WebCore/platform/mock/MockRealtimeVideoSource.cpp

```diff
@@ -573,25 +573,25 @@ void MockRealtimeVideoSource::drawText(GraphicsContext& context)
         string = makeString("Preset size: ", captureSize.width(), " x ", captureSize.height());
         context.drawText(drawingState.statsFont(), TextRun((StringView(string))), statsLocation);
```

Comment:
This has two many parentheses in `TextRun` and uses `StringView` for no obvious reason.